Deeploy/TilingExtension/TilerModel.py

```
@@ -382,9 +387,11 @@ def _solveModel(
 
         timeLimit = self._model.TimeLimit(_SOLVERTIMEOUT)
 
-        log = self._model.SearchLog(1000000)
+        searchLog = self._model.SearchLog(1000000)
```

Ruff (0.12.2)

Local variable `searchLog` is assigned to but never used

Remove assignment to unused variable `searchLog`

(F841)